sohyun/FastFood_sohyun.ipynb

```
@@ -0,0 +1,255 @@
+    "# 한글 폰트 사용을 위한 라이브러리입니다.\n",
+    "!apt-get install -y fonts-nanum"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "id": "f5fa4f19",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# visualization\n",
```

Ruff (0.12.2)

`pickle` imported but unused

Remove unused import: `pickle`

(F401)